.github/workflows/ci.yaml

```diff
@@ -16,3 +19,9 @@ jobs:
       - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
       - run: renovate --version
       - run: renovate-config-validator default.json
+
+  statnett-ci:
+    uses: statnett/github-workflows/.github/workflows/ci.yaml@main
```

Comment:
I don't think this workflow is designed as a reusable one.

https://github.com/statnett/github-workflows/blob/main/.github/workflows/ci.yaml#L4-L8

We should probably make a simple doc, as GH must have all workflows in the same dir.